server/user.go

```diff
@@ -392,3 +392,8 @@ func httpGetUserInfo(w http.ResponseWriter, r *http.Request, p *Plugin) {
 	w.Header().Set("Content-Type", "application/json")
 	_, _ = w.Write([]byte(string(b)))
 }
+
+func (p *Plugin) hasChannelAccess(userID, channelID string) bool {
```

Comment:
Can we specify type for `userID`?

Reply:
@avas27JTG A dedicated type for `userID` is not much useful here. We don't have a dedicated function for UserID.